### DISCUSSION TITLE
坐标量算高度为0

### DISCUSSION THREAD

**Discussion by webdq** [zouyaoji/vue-cesium#336] [Q&A]
加载了地形，坐标量算在有地形上的高度一直显示是0，是只能标记在3D瓦片上才有高度吗？

**Comment by zouyaoji** [comment]
试了一下，能复现，可能有问题，具体等我晚上看看。

**Reply by zouyaoji** [reply]
默认是相对地表的高度~~

pointMeasurementOpts 选项中有个 heightReference参数 ，默认值是1，将它改成 0 即可。

**Reply by layogo** [reply]
改成0了量算出来还是0

**Reply by zouyaoji** [reply]
麻烦你你抽空写个在线的能复现的例子吧

**Reply by layogo** [reply]
![image](https://github.com/zouyaoji/vue-cesium/assets/674578/4654dcac-999c-4635-b7f2-73b78ff076dd)
官网demo的也有问题

**Reply by layogo** [reply]
解决了，就是`pointMeasurementOpts` 这个里面需要设置下：`heightReference: 0,`，示例有两个，我只改了一个
